scripts/install-aws.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 echo "=====Starting installing AWS-CNI ========="
-sed -i s/__VETHPREFIX__/${AWS_VPC_K8S_CNI_VETHPREFIX:-"eni"}/g /app/aws.conf
+sed -i s/__VETHPREFIX__/${AWS_VPC_K8S_CNI_VETHPREFIX:-"eni"}/g /app/10-aws.conflist
 cp /app/aws-cni /host/opt/cni/bin/
 cp /app/aws-cni-support.sh /host/opt/cni/bin/
-cp /app/aws.conf /host/etc/cni/net.d/
+cp /app/10-aws.conflist /host/etc/cni/net.d/
```

Comment:
Can you remove `aws.conf` if it already exists? The file will already be there on existing systems where this is an upgrade.

```bash
if [[ -f /host/etc/cni/net.d/aws.conf ]]; then
    rm /host/etc/cni/net.d/aws.conf
fi
```

Reply:
@micahhausler sure. I can submit it in a second.

Reply:
@micahhausler done. Let me know if you want me to comment it somehow.